src/hls-adapter.js

```diff
@@ -230,6 +232,9 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
     this._hls = new Hlsjs(this._config.hlsConfig);
     this._capabilities.fpsControl = true;
     this._hls.subtitleDisplay = this._config.subtitleDisplay;
+    this._mediaAttachedPromise = new Promise(resolve => {
```

Comment:
I think this is more suitable in `addBindings`, like we do for `_onAddTrack` for example